### PR TITLE
Mention pipx in the documentation

### LINKED ISSUE #248: Mention pipx
Since PyScaffold's `putup` command might be used from any virtual environment or even when no virtual environment is activated, it makes sense to mention [pipx](https://pipxproject.github.io/pipx/) in PyScaffold's documentation. This tool allows you to install putup systemwide in a non-invasive way. We should also add this to our unit tests.

### PR DESCRIPTION
**Pull request by abravalheri**
Closes #248.

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -43,10 +45,31 @@ If you want to install PyScaffold with all official extensions, run::
...
-PyScaffold is also available at `conda-forge`_ and thus can be installed with `conda`_::
+
+Alternative Methods
+===================
+
+It is very easy to get PyScaffold installed with `pip`_, but some people do
+prefer other package managers such as `conda`_ while doing their work.
+
+If you do lots of number crunching or data science in general [#inst2]_ and you already
+rely on `conda-forge`_ packages, you might be attempt the following method::
```
Comment (FlorianWilhelm) [inline docs/install.rst:56]: `be attempt` sounds wrong. `you might also use` seems to be correct.

File: docs/install.rst [hunk 2]
```diff
@@ -43,10 +45,31 @@ If you want to install PyScaffold with all official extensions, run::
...
 
     conda install -c conda-forge pyscaffold
 
+It is also very common for developers to have more then one Python version
+installed in their machines, and a plethora of virtual environments spread all
```
Comment (FlorianWilhelm) [inline docs/install.rst:61]: `in` -> `on`

File: docs/install.rst [hunk 3]
```diff
@@ -43,10 +45,31 @@ If you want to install PyScaffold with all official extensions, run::
...
+over the place… Instead of constantly re-installing PyScaffold in each one of
+these Pythons end venvs, you can use `pipx`_ to do a "minimally-invasive"
```
Comment (FlorianWilhelm) [inline docs/install.rst:63]: I would rather say virtual environments or introduce it as an abbreviation before.

File: docs/install.rst [hunk 4]
```diff
@@ -43,10 +45,31 @@ If you want to install PyScaffold with all official extensions, run::
...
+system-wide installation and have the ``putup`` command always available
+independently of which python you are using::
```
Comment (FlorianWilhelm) [inline docs/install.rst:65]: Python capitalized and you rather mean `virtual environment` here.

**Comment by abravalheri** [comment]
Thank you very much @FlorianWilhelm

**Pull request merged**